### PR TITLE
Install

### PR DESCRIPTION
**Pull request by doutriaux1**
Please merge if you feel it improves your cool software.
More to come if you're interested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,41 @@ Rename `_config.json` to `config.json` and specify the following parameters:
...
+
+#### Regular install
+
+bash ./install.bash
+
+#### Azure
+
+bash ./install.bash azure
+
+### Using CONDA
```
[the remark this answers is not in the document]
Comment (doutriaux1) [reply README.md]: conda keeps everything clean, tight together, no need to add "bloatware" to the system Python.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,41 @@ Rename `_config.json` to `config.json` and specify the following parameters:
...
+
+### Using the script
+
+Go to any directory. Installation will be done in the directory you're running the script from.
+
+#### Regular install
+
+bash ./install.bash
+
+#### Azure
```
[the remark this answers is not in the document]
Comment (doutriaux1) [reply README.md]: Less dependencies, I think some of them where not on conda or in standard conda channels, the idea is also to eventually add other *clouds* and let the user decide which one he's interested.

File: scripts/start_foggycam.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+"""Module to start FoggyCam processing."""
+import argparse
+import json
+from collections import namedtuple
+from foggycam import FoggyCam
+
+print('Welcome to FoggyCam 1.0 - Nest video/image capture tool')
+
+parser = argparse.ArgumentParser("Utilities for NEST CAM")
```
[the remark this answers is not in the document]
Comment (doutriaux1) [reply scripts/start_foggycam.py]: yes to all, start with config file but can overwrite any aspect of it via command line.

**Comment by charlesdoutriaux** [comment]
Would be nice to merge this in.

**Comment by dend** [comment]
Thank you for the contribution! I've just released a new version of the application, and this PR is no longer applicable. For future pull request, I recommend opening an issue first so we can discuss enhancements before those can be implemented.

**Pull request closed without merging**